src/schema/SearchableItem/SearchableItemPresenter.ts

```diff
@@ -0,0 +1,119 @@
+import moment from "moment"
+import { stripTags } from "lib/helpers"
+
+const DATE_FORMAT = "MMM Do, YYYY"
+
+export class SearchableItemPresenter {
+  private item: any
```

Comment:
#minor this can possibly be better typed

Reply:
Agreed! I ran into some trouble using an imported `SearchableItem` type here. Not sure if I had to define or use something else in its place. Would be happy to pair on that sometime.

Reply:
Oh yea, I don't think `typeof SearchableItem` would work, b/c there we define fields like `displayLabel`, and stuff.

Whereas `item` here, is an item of JSON coming from gravity: https://github.com/artsy/gravity/blob/9e12c39f553995f7875406f63caddd33401c70ca/app/api/util/search_helpers.rb#L48

so was thinking can just do:

``` js
type SearchItemRawResponse {
  display: string,
  location: string
}

private item: SearchItemRawResponse
```

etc. to mirror the grav response.